test/gulp-environments.js

```diff
@@ -1,9 +1,14 @@
 var environments = require("../");
 var expect = require("chai").expect;
+var refresh = function(path) {
+  delete require.cache[require.resolve(path)];
+  return require(path);
+}
 
 describe("gulp-environments", function() {
-  var development = environments.make("development");
-  var production = environments.make("production");
+  beforeEach(function() {
+    delete process.env.NODE_ENV;
```

Comment:
We should probably clean up the `process.argv` here as well.

Reply:
Done (see immediately below).